.github/workflows/ci.yml

```diff
@@ -17,3 +17,65 @@ jobs:
       - run: npm run test:lint
         if: matrix.os != 'windows-latest'
       - run: npm run test:unit
+
+  docker:
+    name: Build and test Docker
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: read
+    env:
+      REGISTRY: ghcr.io
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: "arm64"
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build and load Docker image
+        id: docker_build
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          load: true
+          tags: core-test
+          cache-from: type=registry,ref=ghcr.io/filecoin-station/core
+          cache-to: type=inline
+
+      - name: Build Docker image for other platforms
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          platforms: linux/arm64
+          cache-from: type=registry,ref=ghcr.io/filecoin-station/core
+          cache-to: type=inline
+
+      - name: Start Station Core container
+        run: |
+         docker run \
+           --name station \
+           --detach \
+           --env FIL_WALLET_ADDRESS=f1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za \
+           $IMAGEID
+        env:
+          IMAGEID: ${{ steps.docker_build.outputs.imageid }}
+
+      - name: Print station activity
+        run: |
+          sleep 10 # Wait for Station modules to start
+          docker exec station bin/station.js activity
+
+      - name: Check | Saturn started
+        run: docker exec station bin/station.js activity | grep "Saturn module started"
```

Comment:
When we add a new module, we should add another step to check that the new module started too.

For example:

```
      - name: Check | Zinnia started
        run: docker exec station bin/station.js activity | grep "Module Runtime started"
```

@juliangruber FYI